xAPI.md

```diff
@@ -510,6 +513,63 @@ Clarifications and additional examples including:
 - Addiitional explanitory text and diagrams
 [1.0.2...1.0.3](https://github.com/adlnet/xAPI-Spec/compare/1.0.2...1.0.3)
 
+
+<a name="Appendix1B"/>
+### Appendix B: cmi5 Example 
+
+The following example from the cmi5 Community of Practice demonstrates Extensions, "category", etc.
+
+```
+{
+  "id":"2a41c918-b88b-4220-20a5-a4c32391a240",
+  "actor": {
```

Comment:
Not vital, but if you end up making other changes, you could give the actor a name.